### DISCUSSION TITLE
Is there a corresponding UI library？

### DISCUSSION THREAD

**Discussion by Jankos-Lee** [MithrilJS/mithril.js#2703] [Q&A] [answered]
Is there a corresponding UI library？

**Comment by osban** [comment, marked as answer]
There are probably more, but I can think of [Polythene](https://github.com/ArthurClemens/polythene) (Material Design) and [Construct-UI](https://github.com/vrimar/construct-ui).

**Comment by ArthurClemens** [comment]
It is quite trivial to use a HTML-based UI library in Mithril. This is an [example using Semantic-UI](https://flems.io/#0=N4IgZglgNgpgziAXAbVAOwIYFsZJAOgAsAXLKEAGhAGMB7NYmBvEAXwvW10QICsEqdBk2J4s+LLQCuDABQATWtSk4G+AEa15ATwoACYAB00evQDcIMAO6I9sgJR6AvAD49yY6dNZZhkOqliYno-fSMaKAw4OD9EPykIPQAHACcILAwU7T0AoJC2fU8vUz89AGUMMxg9PyLTe0KTLx8-XOC0UIM-akjo2PjEtvz2Gqbi0oARCDhqTPkakDq9eyKAXWNWFbRKEDgYWGpiCHoEHgBGAAZEC4LOHDx8amidoUZmHk0dLqakjHl5CBoADmtgATBckgAPADcGx2ewORxOeDOiDOADY2KsqFBAQBrU6oECYe48LAQYiENLkKhSFLkHgkYhJOCIAD0bJkSTxQMetCwbPJlOpAAFQfgLvgACyCilU6ASQH4fg7YjaJLcXbUNJJUTsO6avYZBgQaiKtCPZ60+l4Jks9ls6jyND8R5QaTyMCRFIwPkCjC8DCQtm49RwNlGjAm6gAWgSbPFUvwZwjMGNRzN5ItTwEIDVGrwcDVsCxrCAA).

Use the [template converter](https://arthurclemens.github.io/mithril-template-converter/) to convert HTML to Mithril syntax.

**Comment by panoply** [comment]
You also have [Mithril Materialized](https://erikvullings.github.io/mithril-materialized/index.html#!/home) by @erikvullings. He also maintains several really helpful UI related tools and of course all good stuff from @ArthurClemens.
